### PR TITLE
Fabo/extension ci fixes

### PR DESCRIPTION
**Pull request by faboweb**

### CODE REVIEW

File: extension/jest.config.js [hunk 1]
```diff
@@ -3,29 +3,29 @@ module.exports = {
   verbose: true,
   testMatch: [`<rootDir>/test/unit/**/*spec.js`],
   transform: {
-    '^.+\\.(js|jsx)?$': `<rootDir>/node_modules/babel-jest`,
-    '.*\\.vue$': `<rootDir>/node_modules/vue-jest`
+    '.*\\.vue$': `vue-jest`,
+    '^.+\\.(js|jsx)?$': `babel-jest`
```
Comment (Bitcoinera) [inline extension/jest.config.js:7]: What is the idea here? Are we going then to have a global `node_modules`? Isn't it better to preserve modularity?

Using `yarn workspace` it is so easy. I would go for separate node_modules

Comment (faboweb) [reply extension/jest.config.js:7]: we are already using yarn workspaces

**Comment by faboweb** [comment]
🚀 Deployed on https://5ede6dc4c9be27023e177062--lunieio.netlify.app

File: extension/jest.config.js (hunk 1 above)
Comment (Bitcoinera) [reply extension/jest.config.js:7]: But your idea here is to have these two modules shared by all the subrepos right?

Comment (faboweb) [reply extension/jest.config.js:7]: I guess though

File: .github/workflows/extension_publish.yml [hunk 2]
```diff
@@ -26,12 +26,12 @@ jobs:
...
-          APP_ID: ${{ secrets.EXTENSION_APP_ID }}
+      # - name: Deploy
+      #   run: sh ./scripts/deploy.sh ./dist-zip/lunie-browser-extension.zip
+      #   working-directory: ${{env.WORKING_DIRECTORY}}
+      #   env:
+      #     GAPI_CLIENT_ID: ${{ secrets.GAPI_CLIENT_ID }}
+      #     GAPI_CLIENT_SECRET: ${{ secrets.GAPI_CLIENT_SECRET }}
+      #     GAPI_REFRESH_TOKEN: ${{ secrets.GAPI_REFRESH_TOKEN }}
+      #     ACCESS_TOKEN: ${{ secrets.GAPI_ACCESS_TOKEN }}
+      #     APP_ID: ${{ secrets.EXTENSION_APP_ID }}
```
Comment (Bitcoinera) [inline .github/workflows/extension_publish.yml:37]: What is the GAPI_CLIENT? Google API?

Comment (faboweb) [reply .github/workflows/extension_publish.yml:37]: yep, for publishig the extension

Comment (Bitcoinera) [reply .github/workflows/extension_publish.yml:37]: Thanks

File: extension/jest.config.js (hunk 1 above)
Comment (Bitcoinera) [reply extension/jest.config.js:7]: why make an exception here? Better have them always separate except when needed some meta-module, like in [this example](https://github.com/ferreiro/example-monorepo/blob/master/package.json) with concurrently, no?

To keep the same logic

Comment (faboweb) [reply extension/jest.config.js:7]: is there any problem right now with the logic?

File: extension/src/messageHandlers.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ const {
 } = require('@lunie/cosmos-keys')
 const {
   default: TransactionManager
-} = require('../lunie/src/signing/transaction-manager')
+} = require('app/src/signing/transaction-manager')
```
Comment (Bitcoinera) [inline extension/src/messageHandlers.js:8]: This means we don't have the submodule there right? 🤩 

Monorepo all the way! This is such a massive improvement! 💃

File: extension/jest.config.js (hunk 1 above)
Comment (Bitcoinera) [reply extension/jest.config.js:7]: well, I would think so by looking at this. But maybe this is the same than with the `rootDir`, just stating `vue-jest`?

**Pull request merged**